tensorflow/training/docker/2.0.2/py2/Dockerfile.gpu

```diff
@@ -94,6 +94,10 @@ RUN echo "hwloc_base_binding_policy = none" >> /usr/local/etc/openmpi-mca-params
 RUN echo NCCL_DEBUG=INFO >> /etc/nccl.conf
 
 ENV LD_LIBRARY_PATH=/usr/local/openmpi/lib:$LD_LIBRARY_PATH
+# Resolve missing libcuda.so issue
+RUN cp /usr/local/cuda-10.1/extras/CUPTI/lib64/libcupti* /usr/local/cuda/lib64/
```

Comment:
can you verify the libcupti.so's path by `find / -name libcupti*` in the container? I don't think 2.0 container has 10.1 libs

Reply:
Thanks for reviewing. I will follow this suggestion to do local build and test first.